docs/cse/index.md

```diff
@@ -1,12 +1,12 @@
 ---
 slug: /cse
-title: Cloud SIEM Enterprise
+title: Cloud SIEM
 tags: [cse, cloud siem, security]
```

Comment:
Can we remove this CSE tag too?

Reply:
Yes we can, @JV0812. I missed that one. Fixed now.